src/scripts/generate-crop-table.js

```diff
@@ -0,0 +1,91 @@
+/** @typedef {import("../index").farmhand.item} farmhand.item */
+/** @typedef {import("../index").farmhand.cropVariety} farmhand.cropVariety */
+
+// Must be invoked with babel: https://stackoverflow.com/a/51532127/470685
+import markdownTable from 'markdown-table'
+
+import { levels } from '../data/levels.js'
+import { itemsMap } from '../data/maps'
+import { moneyString } from '../utils/moneyString.js'
+
+const getDaysToMature = seedItem => {
+  return Object.values(seedItem.cropTimetable).reduce(
+    (days, acc) => days + acc,
+    0
+  )
+}
+
+/**
+ * @param {farmhand.item} seedItem
+ * @param {(farmhand.item|farmhand.cropVariety)} cropItem
+ */
+function getCropImage(seedItem, cropItem) {
+  if (Array.isArray(seedItem.growsInto)) {
+    return `![${cropItem.name}](https://raw.githubusercontent.com/jeremyckahn/farmhand/main/src/img/items/${cropItem.imageId}.png)`
+  } else {
+    return `![${cropItem.name}](https://raw.githubusercontent.com/jeremyckahn/farmhand/main/src/img/items/${cropItem.id}.png)`
+  }
+}
+
+/**
+ * @param {farmhand.item} seedItem
+ */
+function getSeedImage(seedItem) {
+  return `![${seedItem.name}](https://raw.githubusercontent.com/jeremyckahn/farmhand/main/src/img/items/${seedItem.id}.png)`
+}
+
+const headers = [
+  'Crop',
+  'Seed',
+  'Unlocked at level',
+  'Base value',
+  'Days to mature',
+  'Tier',
+]
+
+const rows = []
+
+/**
+ * @param {number} level
+ * @param {farmhand.item} seedItem
+ * @param {farmhand.item} cropItem
+ */
+const getCropRow = (level, seedItem, cropItem) => {
+  return [
+    `${getCropImage(seedItem, cropItem)} ${cropItem.name}`,
+    `${getSeedImage(seedItem)} ${seedItem.name}`,
+    level,
+    moneyString(seedItem.value),
+    getDaysToMature(seedItem),
+    seedItem.tier,
+  ]
+}
+
+for (const level of levels) {
```

Comment:
what do you think about doing this with `map` instead? that way you don't have to push onto `rows` but can instead do `const rows = levels.map((level) => {`?

Reply:
There's one issue with that approach: This isn't a straight mapping. There's a nested loop at line 75 that handles crops with varieties. This code could be rewritten to use a `map`, but on balance it seems like it would have roughly equivalent complexity.

Reply:
ahh i overlooked that. sounds good then!